Vehicle/AppInfo.h

```diff
@@ -22,7 +22,7 @@
 #define kOrganizationDomain             "esri.com"
 
 #define kApplicationName                "DSA_Vehicle_Qt"
-#define kApplicationVersion             "1.1.6"
+#define kApplicationVersion             "2.0.0"
```

Comment:
Is there some DSA versioning we should follow? 2.0.0 seems like a good choice.

Reply:
Not that I'm aware of. I suppose you could make an argument for this just getting updated to 1.2.0 since the app itself doesn't have much new functionality. Maybe that would make more sense.

Reply:
I think 2.0.0 makes sense. It brings it up to both Qt 6 and Qt Maps SDK 200.0, which are both major-version releases.